src/StudioComponents/Util/getDragInput.lua

```diff
@@ -167,5 +167,5 @@ return function(props: DragInputProperites): (vector2Value, types.Computed<Vecto
 		end
 	end))
 
-	return currentValue, currentAlpha
-end
\ No newline at end of file
+	return currentValue, currentAlpha, isDragging
```

Comment:
the types returned by this function should be updated to:
`(vector2Value, types.Computed<Vector2>, types.Value<boolean>)`